packages/vega-runtime/src/context.js

```diff
@@ -65,7 +65,10 @@ Context.prototype = Subcontext.prototype = {
     // wipe out targets first for better efficiency
     const keys = Object.keys(ctx.nodes);
     for (const key of keys) ctx.nodes[key]._targets = null;
-    for (const key of keys) ctx.nodes[key].detach();
+    for (const key of keys) {
+      ctx.nodes[key].detach();
+      delete ctx.nodes[key];
```

Comment:
Is the `delete` operation really necessary? (We null out `ctx.nodes` right after this.) Or is there a remaining pointer to that object elsewhere? If so, we might want address that directly...

If it does make an improvement to delete, we might instead assign `undefined`, as this is faster than actually removing the property.

Reply:
Sorry, I can't remember... I think that there may have been references to it held by forked subcontexts, but I could be wrong.

Reply:
I tried removing this change and rechecking the issue in my dev environment and it doesn't look to be necessary.